example/chunks/README.md

```diff
@@ -0,0 +1,849 @@
+# Snapshots API examples
```

Comment:
It seems better to add few examples in `example/snapshots/README.md` for chunks as well.
Pls delete `example/chunks` folder, add the following few examples showing chunks in `example/snapshots/README.md`:

```
GET https://api.enterprise.wikimedia.com/v2/snapshots/hiwiki_namespace_0

{
    "identifier": "hiwiki_namespace_0",
    "version": "7905d6e5bcf9714a467353aa2527fe12",
    "date_modified": "2024-08-30T00:02:34.395793938Z",
    "is_part_of": {
        "identifier": "hiwiki"
    },
    "in_language": {
        "identifier": "hi"
    },
    "namespace": {
        "identifier": 0
    },
    "size": {
        "value": 769.714e0,
        "unit_text": "MB"
    },
    "chunks": [
        "hiwiki_namespace_0_chunk_0",
        "hiwiki_namespace_0_chunk_1",
        "hiwiki_namespace_0_chunk_2"
    ]
}

GET https://api.enterprise.wikimedia.com/v2/snapshots/hiwiki_namespace_0/chunks/1


{
    "identifier": "hiwiki_namespace_0_chunk_1",
    "version": "6fbc6bf3fce195b356dfe44af7fbb540",
    "date_modified": "2024-08-30T00:02:16.265448519Z",
    "is_part_of": {
        "identifier": "hiwiki"
    },
    "in_language": {
        "identifier": "hi"
    },
    "namespace": {
        "identifier": 0
    },
    "size": {
        "value": 276.067712e0,
        "unit_text": "MB"
    }
}